bin/cylc-run

```diff
@@ -95,7 +96,11 @@ warm start unless you use special cold-start tasks. See "Suite Start-up" and
             start_point_string = self.args[1]
             if start_point_string == "now":
                 start_point_string = (
-                    datetime.datetime.utcnow().strftime("%Y%m%dT%H%MZ")
+                    datetime.datetime.now().strftime("%Y%m%dT%H%M")
+                )
+            elif start_point_string == "UTCnow":
+                start_point_string = (
+                    datetime.datetime.utcnow().strftime("%Y%m%dT%H%M")
```

Comment:
Ah, I'm not sure it doeZ ...   I originally removed the Z for the wrong reason (my problem turned out to be caused by the ':' issue above), BUT by testing with ```initial cycle point = UTCnow``` in both ```UTC mode = True``` and not, the presence of a Z here makes no difference.  Although it presumably should!